mathesar/imports/base.py

```diff
@@ -11,7 +11,7 @@
 
 
 def create_table_from_data_file(data_file, name, schema, comment=None):
-    engine = create_mathesar_engine(schema.database.name)
+    engine = schema._sa_engine
```

Comment:
@mathemancer quick look? @pavish told me this PR worsened the Postgres connection bottleneck; after a quick look around I found that our `mathesar` layer tests rely on this routine a lot, which doesn't use the engine cache. I'm hopeful that this will improve resource usage in tests outside this PR as well.

Reply:
I think it may have had something to do with sanitizing the engine. However, it may just be a general oversight compounded by pattern-following. I have no problem with making this change.

Reply:
Thanks @dmos62 and @mathemancer. I'm going to go ahead and merge the PR in.